sentry_sdk/integrations/starlette.py

```diff
@@ -456,6 +468,48 @@ def event_processor(event, hint):
     starlette.routing.request_response = _sentry_request_response
 
 
+def patch_templates():
+    # type: () -> None
+
+    # If markupsafe is not installed, then Jinja2 is not installed
+    # (markupsafe is a dependency of Jinja2)
+    # In this case we do not need to patch the Jinja2Templates class
+    try:
+        from markupsafe import Markup
+    except ImportError:
+        Markup = None  # type: ignore  # noqa: N806
```

Comment:
Maybe we can just early `return` here and then we don't need the `if Markup is not None` check below

Feels a bit weird to check if `jinja2` is installed by checking for its dependency, I'm guessing we do it like this because we want to use `Markup` below?

Reply:
Yes, that is the reason. But good idea with returning.

Reply:
flat is better than nested :-)